thumbnail-service/src/test/java/no/sikt/nva/thumbnail/ThumbnailRequestHandlerTest.java

```diff
@@ -56,19 +59,44 @@ class ThumbnailRequestHandlerTest {
     private static final String ALREADY_HAVE_SIZE_AS_THUMBNAIL_OUTPUT = "correct-size.png";
     private static final String THUMBNAIL_URI_TEMPLATE_STRING = "https://%s.s3.%s.amazonaws.com/%s";
     private static final String THUMBNAIL_BUCKET_NAME = "dlr-nva-thumbnails";
+    private static final String QUICK_TIME_MOVIE_FILENAME = "quickTimeWindow.mov";
+    private static final String QUICK_TIME_MIME_TYPE = "video/quicktime";
+    private static final String MOVIE_PATH = "videos";
     private TestAppender appender;
 
+    private ThumbnailerInitializer thumbnailerInitializer;
+
     @BeforeEach
-    public void init() {
+    public void init() throws IOException {
         this.appender = LogUtils.getTestingAppenderForRootLogger();
+        this.thumbnailerInitializer = new ThumbnailerInitializer.Builder()
+                                          .withFFmpeg(new FakeFFmpeg())
+                                          .withFFprobe(new FakeFFprobe())
+                                          .build();
+    }
+
+    @Test
+    public void shouldBeAbleToConvertQuickTimeMovie() throws IOException {
+        var s3Path = randomS3Path();
+        var expectedThumbnailURL = craftExpectedURL(s3Path);
+        var s3Client = new FakeS3ClientWithPutObjectSupport(QUICK_TIME_MOVIE_FILENAME,
+                                                            MOVIE_PATH,
+                                                            QUICK_TIME_MIME_TYPE);
+        var s3Event = createNewFileUploadEvent(MOVIE_PATH
```

Comment:
consider using Path or UnixPath for making paths:
```
 @Test
    public void shouldBeAbleToConvertQuickTimeMovie() throws IOException {
        var s3Path = randomS3Path();
        var expectedThumbnailURL = craftExpectedURL(s3Path);
        var s3Client = new FakeS3ClientWithPutObjectSupport(QUICK_TIME_MOVIE_FILENAME,
                                                            MOVIE_PATH,
                                                            QUICK_TIME_MIME_TYPE);
        
        var s3Event = createNewFileUploadEvent(UnixPath.of(MOVIE_PATH,QUICK_TIME_MOVIE_FILENAME),
                                               s3Client, s3Path);
        var handler = new ThumbnailRequestHandler(s3Client, thumbnailerInitializer);
        var thumbnailUrl = handler.handleRequest(s3Event, CONTEXT);
        assertThat(thumbnailUrl, is(equalTo(expectedThumbnailURL)));
    }
    
    private S3Event createNewFileUploadEvent(UnixPath path,
                                             FakeS3ClientWithPutObjectSupport s3Client,
                                             UnixPath s3Path) throws IOException {
        var s3Driver = new S3Driver(s3Client, INPUT_BUCKET_NAME);
        var insertStream = IoUtils.inputStreamFromResources(path.toString());
        var uri = s3Driver.insertFile(s3Path, insertStream);
        return createS3Event(uri);
    }
```

Reply:
I prefer Path although the method seems as deprecated in commons. We will de-deprecate it

Reply:
Fixed